package.json

```diff
@@ -11,7 +11,7 @@
     "lint": "npm run lint6 && npm run lint-ts && npm run lint5 && npm run lint-tests",
     "lint6": "eslint -c .eslintrc.es6.json --max-warnings 0 'assets/js/src/**/*.jsx' 'tests/javascript/**/*.js'",
     "lint-ts": "eslint -c .eslintrc.ts.json --max-warnings 0 'assets/js/src/**/*.tsx' 'assets/js/src/**/*.ts'",
-    "lint5": "eslint -c .eslintrc.es5.json --max-warnings 0 'assets/js/src/**/*.js' 'webpack.config.js'",
+    "lint5": "eslint -c .eslintrc.es5.json --max-warnings 0 'assets/js/src/**/*.js'",
```

Comment:
Feel free to modify this, but I needed the spread operator in the webpack config to support our build. These linting rules were too aggressive.

Reply:
Sorry, for historical reasons we use some old Eslint rules. You could `Object.assign` instead of `...`, couldn't you?